packages/dds/merge-tree/package.json

```diff
@@ -98,82 +98,13 @@
     "typescript-formatter": "7.1.0"
   },
   "typeValidation": {
-    "version": "1.0.0",
+    "version": "1.1.0",
     "broken": {
-      "ClassDeclaration_BaseSegment": {
-        "forwardCompat": false
-      },
       "ClassDeclaration_Client": {
         "forwardCompat": false
       },
-      "InterfaceDeclaration_IConsensusInfo": {
-        "backCompat": false,
-        "forwardCompat": false
-      },
-      "InterfaceDeclaration_IHierBlock": {
-        "forwardCompat": false
-      },
-      "InterfaceDeclaration_IMergeBlock": {
-        "forwardCompat": false
-      },
-      "TypeAliasDeclaration_IMergeNode": {
-        "forwardCompat": false
-      },
-      "InterfaceDeclaration_IMergeNodeCommon": {
-        "forwardCompat": false
-      },
-      "ClassDeclaration_IncrementalMapState": {
-        "forwardCompat": false
-      },
-      "InterfaceDeclaration_InsertContext": {
-        "forwardCompat": false
-      },
-      "InterfaceDeclaration_IRemovalInfo": {
-        "forwardCompat": false
-      },
-      "InterfaceDeclaration_ISegmentChanges": {
-        "forwardCompat": false
-      },
       "ClassDeclaration_LocalReference": {
```

Comment:
@ransomr see earlier comment. Just FYI, you may be a better judge of this was expected.